NEWS

```diff
@@ -18,6 +18,20 @@ CHANGES WITH 253 in spe:
           For more details, see:
           https://lists.freedesktop.org/archives/systemd-devel/2022-September/048352.html
 
+        * We intend to change behaviour w.r.t. user units and sandboxing
+          options, so that they work in the same way as system units by default.
+          This requires implicitly enabling user namespaces (PrivateUsers=yes)
+          when a sandboxing option is enabled in a user unit. The drawback is
+          that system users will no longer be visible (and apper as 'nobody') to
+          the user unit. By definition a sandboxed user unit should run with no
+          privileges, so impact should be minimal. This will remove a great
+          source of confusion that has been reported by users over the years,
+          due to how system and user units behave differently on the same system
+          with the same options. We plan to enable this change in the next
```

Comment:
well, this is misleading. it suggests they work the same now, as above. but they work quite differently. 

they have an effect at all now, that's the key takeaway, even if the implementation and behaviour regarding other UIDs differs.